wazo_confd/plugins/line/resource.py

```diff
@@ -58,5 +63,5 @@ def put(self, id):
     def delete(self, id):
         tenant_uuid = build_tenant()
         tenant_uuids = self._build_tenant_list({'recurse': True})
-        self._middleware.delete(id, tenant_uuid, tenant_uuids)
+        self._middleware.delete(id, tenant_uuid, tenant_uuids, recursive=is_recursive())
```

Comment:
Does the recursive option bypass protection of associated extensions? Will it only delete associated extensions that would be orphaned?

Reply:
It deletes only the extensions if not used at all. I added 2 tests : https://github.com/wazo-platform/wazo-confd/blob/1dab14d800e55f52d18cba2531eb6621626c5e90/integration_tests/suite/base/test_lines.py#L369

Reply:
Oh thanks!